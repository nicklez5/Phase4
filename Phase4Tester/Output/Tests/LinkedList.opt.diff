--- SelfTestCases/LinkedList.opt.out	2011-04-28 14:55:21.000000000 -0700
+++ ./Output/Tests/LinkedList.opt.out	2020-03-14 13:04:32.301787203 -0700
@@ -1,27 +1,2 @@
-25
-10000000
-39
-25
-10000000
-22
-39
-25
-1
-0
-10000000
-28
-22
-39
-25
-2220000
--555
--555
-28
-22
-25
-33300000
-22
-25
-44440000
-0
+null pointer
 
