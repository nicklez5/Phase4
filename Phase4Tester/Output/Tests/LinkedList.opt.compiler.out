.data

empty_Element:

empty_List:

empty_LL:

.text

  jal Main
  li $v0 10
  syscall

Main:
  sw $fp -8($sp)
  move $fp $sp
  subu $sp $sp 8
  sw $ra -4($fp)
  la $a0 empty_LL
  jal LL.Start
  move $t0 $v0
  move $a0 $t0
  jal _print
  lw $ra -4($fp)
  lw $fp -8($fp)
  addu $sp $sp 8
  jr $ra

Element.Init:
  sw $fp -8($sp)
  move $fp $sp
  subu $sp $sp 8
  sw $ra -4($fp)
  move $t0 $a0
  move $t1 $a1
  move $t2 $a2
  move $t3 $a3
  move $t0 $v0
